app/assets/javascripts/discourse/app/widgets/header.js

```diff
@@ -331,6 +333,24 @@ export function attachAdditionalPanel(name, toggle, transformAttrs) {
   additionalPanels.push({ name, toggle, transformAttrs });
 }
 
+createWidget("revamped-hamburger-menu-wrapper", {
+  buildAttributes() {
+    return { "data-click-outside": true };
+  },
+
+  html() {
+    return [
+      new ComponentConnector(this, "hamburger-menu-wrapper", {}, [], {
```

Comment:
I think you're already aware but we now have a better way to render a Glimmer component inside widgets that doesn't require a wrapper classic component, see https://github.com/discourse/discourse/commit/98bacbd2c6b9fe57167cd32af5eb4839b4a5d1f6 and https://github.com/discourse/discourse/commit/6c5efb61c992fb9850f900b49f5df6ca57bfff9a.